<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>superusuario</class>
 <widget class="QMainWindow" name="superusuario">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1080</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1080</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="titulo_menu_su">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>1081</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Decotura ICG Inline</family>
      <pointsize>50</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family: Decotura ICG Inline;
font-size: 50pt;</string>
    </property>
    <property name="text">
     <string>Menú super usuario</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_salir">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>50</y>
      <width>111</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Decotura ICG</family>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family: Decotura ICG;
font-size: 30pt;
background-color: #ff0000;
border-radius: 20px;
color: white;</string>
    </property>
    <property name="text">
     <string>Salir</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_pacientes">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>190</y>
      <width>381</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Decotura ICG</family>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family: Decotura ICG;
font-size: 30pt;
background-color: #00aaff;
border-radius: 20px;
color: white;</string>
    </property>
    <property name="text">
     <string>Pacientes</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_citas">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>310</y>
      <width>381</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Decotura ICG</family>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family: Decotura ICG;
font-size: 30pt;
background-color: #00aaff;
border-radius: 20px;
color: white;</string>
    </property>
    <property name="text">
     <string>Citas</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_facturas">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>430</y>
      <width>381</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Decotura ICG</family>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family: Decotura ICG;
font-size: 30pt;
background-color: #00aaff;
border-radius: 20px;
color: white;</string>
    </property>
    <property name="text">
     <string>Facturas</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_estudios">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>180</y>
      <width>381</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Decotura ICG</family>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family: Decotura ICG;
font-size: 30pt;
background-color: #00aaff;
border-radius: 20px;
color: white;</string>
    </property>
    <property name="text">
     <string>Estudios</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_laboratoristas">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>430</y>
      <width>381</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Decotura ICG</family>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family: Decotura ICG;
font-size: 30pt;
background-color: #00aaff;
border-radius: 20px;
color: white;</string>
    </property>
    <property name="text">
     <string>Laboratoristas</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_medicos">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>310</y>
      <width>381</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Decotura ICG</family>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family: Decotura ICG;
font-size: 30pt;
background-color: #00aaff;
border-radius: 20px;
color: white;</string>
    </property>
    <property name="text">
     <string>Médicos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_usuarios">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>550</y>
      <width>381</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Decotura ICG</family>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family: Decotura ICG;
font-size: 30pt;
background-color: #00aaff;
border-radius: 20px;
color: white;</string>
    </property>
    <property name="text">
     <string>Usuarios</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
